posthog/test/test_dbrouter.py

```diff
@@ -0,0 +1,14 @@
+from posthog.dbrouter import ReplicaRouter
+from posthog.test.base import BaseTest
+from posthog.models.user import User
+from posthog.models.team import Team
+
+
+class TestReplicaRouter(BaseTest):
+    def test_opted_in_models_are_replica_routed(self) -> None:
+        router = ReplicaRouter(["User"])
+
+        self.assertEqual(router.db_for_write(User), "default")
```

Comment:
minor nit: For a second was confused, shouldn't this be 'replica', but then realised it's for writing.

Perhaps rename `ReplicaRouter` to `ReadReplicaRouter`. I hear RRR is very famous 😂